app/frontend/resources/routes.edn

```diff
@@ -236,6 +236,14 @@
           :args {:contract-ids
                  (clojure.string/split contract-ids #"-")}}}
 
+ :partners
+ {:path "/contracts/:contract-ids/partners"
+  :view teet.contract.contract-view/partner-page
+  :crumb (teet.contract.contract-model/contract-url-id state) ;; TODO: fix
+  :title (teet.contract.contract-model/contract-name state) ;; TODO: fix
```

Comment:
TODO comments?

Reply:
TODO comments as schema is not yet available to implement everything